installation/Developer-guide.md

```diff
@@ -0,0 +1,78 @@
+## Logging Bugs
```

Comment:
Looks good, maybe add a title, a small intro, (e.g. something like "This page describes how to test and debug your changes one you have set up a development environement of Product Opener with Docker" and also a link to the installation guide at the top of the document.

Reply:
On it.